app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+import { CONTACTS_INITIAL_FORM_VALUES } from "../constants";
+
+const NewContactPane = ({ showPane, setShowPane, setContacts }) => {
```

Comment:
What's the advantage of having the component name as `NewContactPane` and the filename as an unrelated `Create.jsx`?